contracts/scripts/native_solc_compile_all

```diff
@@ -62,10 +62,10 @@ $SCRIPTPATH/native_solc8_6_compile BatchBlockhashStore.sol
 $SCRIPTPATH/native_solc8_6_compile BatchVRFCoordinatorV2.sol
 
 # VRF V2 Wrapper
-$SCRIPTPATH/native_solc8_15_compile VRFV2Wrapper.sol
```

Comment:
I notice that separate files `native_solc_compile_all_ocr2vrf` and `native_solc8_15_compile_ocr2vrf` still use 0.8.15.

What's the status of these contracts? Are they used in prod right now?

The highest version we fully support atm is 0.8.6, and we should remove any higher versions until we decide otherwise.

Reply:
Great new @makramkd , thank you! Could you please tag me in that ticket?